lib/module.js

```diff
@@ -52,4 +51,5 @@ function applyPrefix(definitionsToAdd) {
   }, {})
 }
 
+module.exports = separateEnvModule
```

Comment:
Keep it ESM syntax please

Reply:
If it is not ESM we open coverage for Nuxt 1x
In my opinion we should release a version for Nuxt 1x and later a major release only covering Nuxt 2x
Conversation https://github.com/nuxt-community/redirect-module/pull/21